WordPress/src/main/java/org/wordpress/android/ui/media/MediaGridAdapter.java

```diff
@@ -240,16 +240,15 @@ public void bindView(final View view, Context context, Cursor cursor) {
 
                 // add onclick to retry failed uploads
                 if (state.equalsIgnoreCase(MediaUploadState.FAILED.name())) {
-                    state = MediaUploadState.QUEUED.name();
                     holder.stateTextView.setVisibility(View.VISIBLE);
-                    holder.stateTextView.setText(state);
+                    holder.stateTextView.setText(context.getString(R.string.retry));
```

Comment:
We should do the same thing above `holder.stateTextView.setText(state);` - I'll fill a new issue